packer/scripts/install-kubernetes.sh

```diff
@@ -4,6 +4,8 @@
 
 set -o errexit -o xtrace
 
+swapoff -a
```

Comment:
Add comment please (something along the lines of "required by kubernetes" is good enough)